kolibri/core/assets/src/views/CoreBase/ScrollingHeader.vue

```diff
@@ -287,16 +134,27 @@
   .scrolling-header {
     @extend %enable-gpu-acceleration;
 
+    position: fixed;
     top: 0;
     right: 0;
     left: 0;
     z-index: 4;
-  }
 
-  .ease {
-    transition-timing-function: ease-in;
-    transition-duration: 0.1s;
-    transition-property: transform;
+    // Use different transition timing functions depending on direction
+    // to maintain some kind of symmetry
+    &.dir-up {
+      transition: top 0.25s ease-in;
```

Comment:
recommend animating with `translate` instead of `top` for performance:

https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/

Also, rather than `ease-in` and `ease-out`, we can consider using Material's curves:

| name | curve |
|--|--|
| Standard easing | `cubic-bezier(0.4, 0.0, 0.2, 1);` |
| Decelerate easing | `cubic-bezier(0.0, 0.0, 0.2, 1);` |
| Accelerate easing | `cubic-bezier(0.4, 0.0, 1, 1);`|

Reply:
Which curves would replace ease-in and ease-out?

Reply:
whoops, forgot to link to the source: https://material.io/design/motion/speed.html#easing

They seem to recommend decelerate (ease out) for incoming (`dir-down`) and accelerate (ease in) for outgoing (`dir-up`).